src/api/types/generated-schema.ts

```diff
@@ -4550,7 +4550,7 @@ export interface components {
             /** Format: int32 */
             germinatingQuantityAdded: number;
             /** Format: int32 */
-            hardeningOffQuantity: number;
+            hardeningOffQuantityAdded: number;
```

Comment:
@ice1080 I think this property needs the `Added` suffix, I manually changed it after regenerating the types.

Reply:
Server PR to fix this, sorry about that 😞 
https://github.com/terraware/terraware-server/pull/3295